<ui version="4.0" >
 <class>WdgToneMappingDialog</class>
 <widget class="QWidget" name="WdgToneMappingDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Operator:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QComboBox" name="comboBoxOperators" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Preset:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QComboBox" name="comboBoxPresets" />
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QWidget" native="1" name="centralWidgetHolder" />
   </item>
   <item row="3" column="2" >
    <widget class="QPushButton" name="pushButtonOk" >
     <property name="text" >
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QPushButton" name="pushButtonCancel" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
